features/aspect-ratio.yml.dist

```diff
@@ -14,6 +14,31 @@ status:
     safari: "15"
     safari_ios: "15"
 compat_features:
-  - css.properties.aspect-ratio
-  - html.elements.img.aspect_ratio_computed_from_attributes
+  # baseline: high
+  # baseline_low_date: 2020-09-16
+  # baseline_high_date: 2023-03-16
+  # support:
+  #   chrome: "79"
+  #   chrome_android: "79"
+  #   edge: "79"
+  #   firefox: "71"
+  #   firefox_android: "79"
+  #   safari: "14"
+  #   safari_ios: "14"
   - html.elements.video.aspect_ratio_computed_from_attributes
+
+  # baseline: high
+  # baseline_low_date: 2021-09-20
+  # baseline_high_date: 2024-03-20
+  # support:
+  #   chrome: "79"
+  #   chrome_android: "79"
+  #   edge: "79"
+  #   firefox: "71"
+  #   firefox_android: "79"
+  #   safari: "15"
+  #   safari_ios: "15"
+  - html.elements.img.aspect_ratio_computed_from_attributes
+
+  # Same status as overall feature
```

Comment:
OK, I'm now thinking that this comment ought to show the full status as well—it's some distance from the status it's referencing, which makes it hard to review (also because there's no diff to the status itself, I have to click to expand it).

What do you think of something like this, so it stands out, too?

```suggestion
  # ⬇️ Same status as overall feature ⬇️
  # baseline: high
  # …
```

Reply:
I've had the same thought, so I'll make it so.

Reply:
Done.